flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1341,26 +1365,26 @@ def test_created_is_listed(self):
 
     def test_listed_volume_attributes(self):
         """
-        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the same
-        dataset_id and size as was passed to ``create_volume``.
+        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the
+        same dataset_id and (maybe over-allocated size) as was passed
+        to ``create_volume``.
         """
         expected_dataset_id = uuid4()
+        requested_size = REALISTIC_BLOCKDEVICE_SIZE
```

Comment:
I can't use ``allocation_unit`` because of the minimum size requirement on Rackspace, but I can request ``allocated_size(allocation_unit, REALISTIC_....)` instead and add that extra test you suggest.
Then a followup issue to supply ``MINIMUM_VOLUME_SIZE`` to the test case factory function...#1406 in fact...and replace `REALISTIC_BLOCKDEVICE_SIZE` throughout.
(sorry @wallnerryan for not contributing to the conversation on that PR)